charts/kserve-resources/templates/clusterservingruntimes.yaml

```diff
@@ -8,10 +8,12 @@ spec:
     prometheus.kserve.io/path: "/metrics"
   supportedModelFormats:
     - name: lightgbm
-      version: "2"
+      version: "3"
```

Comment:
Also need to add the validation webhook to the chart?

Reply:
Yes, I forgot.